tests/unit/command/test_studio.py

```diff
@@ -0,0 +1,91 @@
+from unittest import mock
+
+from dvc_studio_client.auth import AuthorizationExpired
+
+from dvc.cli import main
+from dvc.utils.studio import STUDIO_URL
+
+
+def test_studio_login_invalid_scope():
+    assert main(["studio", "login", "--scopes", "invalid!"]) == 1
+
+
+@mock.patch("dvc_studio_client.auth.get_access_token")
+def test_studio_login_token_check_failed(mock_get_access_token):
```

Comment:
```suggestion
def test_studio_login_token_check_failed(mocker):
	mock = mocker.patch("dvc_studio_client.auth.get_access_token")
```

We use `mocker` fixture instead of using `@mock.patch`. Also it's better to use simpler names like `mock` instead when there is only a single mock.